lib/matplotlib/cbook.pyi

```diff
@@ -74,37 +74,25 @@ def open_file_cm(
 def is_scalar_or_string(val: Any) -> bool: ...
 @overload
 def get_sample_data(
-    fname: str | os.PathLike, asfileobj: Literal[True] = ..., *, np_load: Literal[True]
-) -> np.ndarray: ...
+    fname: str | os.PathLike, asfileobj: Literal[True] = ...
+) -> np.ndarray | IO: ...
```

Comment:
Unions in returns are mildly annoying... but this is an ancillary function anyway, so not too worried.

(Also the old one was not technically accurate, though practically speaking if you passed True, you better be using a valid filetype for numpy loading)

Reply:
Unfortunately, the return value depends on the extension in `fname`, so I don't think there's any easy way to split the union.

Reply:
Yeah, could extend to just `Any`, but that isn't a satisfying answer... I'm okay with it, just noted in case you or someone else wanted to suggest a change based on that.